packages/react-components/react-charts-preview/src/utilities/ChartHoverCard/ChartHoverCard.styles.ts

```diff
@@ -10,7 +10,7 @@ export const getChartHoverCardStyles = (props: IChartHoverCardStyleProps): IChar
         display: 'grid',
         overflow: 'hidden',
         padding: '11px 16px 10px 16px',
-        backgroundColor: theme.semanticColors.bodyBackground,
+        backgroundColor: tokens.colorNeutralBackground1,
```

Comment:
are these changes sufficient to get chart hover card to work?

Reply:
yes, hover card is working. Will just merge it with popover component in next PR

Reply:
you are using v8 dependencies in chart hover card. They need to be removed.

Reply:
yes that is left, haven't started with chart hover card yet.

Reply:
This is the only error that is left during build now

Reply:
ok will start next